homeassistant/components/scene/velux.py

```diff
@@ -38,16 +35,7 @@ def name(self):
         """Return the name of the scene."""
         return self.scene.name
 
-    @property
-    def should_poll(self):
-        """Return that polling is not necessary."""
-        return False
-
-    @property
-    def is_on(self):
-        """There is no way of detecting if a scene is active (yet)."""
-        return False
-
-    def activate(self):
+    @asyncio.coroutine
+    def async_activate(self):
         """Activate the scene."""
-        self.hass.async_add_job(self.scene.run())
+        yield from self.scene.run()
```

Comment:
Is `scene.run` a coroutine?

Reply:
@balloob : yes, https://github.com/Julius2342/pyvlx/blob/master/pyvlx/scene.py#L20